manifests/kiali-community/1.37.0/manifests/kiali.v1.37.0.clusterserviceversion.yaml

```diff
@@ -419,7 +419,6 @@ spec:
           - configmaps
           - endpoints
           - pods/log
-          - pods/proxy
```

Comment:
we already released 1.37.0 and this didn't make it. So remove this change from this 1.37 manifest. I will create a new 1.38 manifest and you can put this change in there. I will do this soon and post back when that is ready

Reply:
Someone approve and merge this: https://github.com/kiali/kiali-operator/pull/358
Then you can revert this line change and make the identical change to the v1.38.0 version of the manifest.